### ISSUE TITLE
Lighting Estimation

### ISSUE THREAD

**Issue by vanlooverenkoen** [google-ar/sceneform-android-sdk#791]
Is it possible to provide us with a sample about lighting estimation? The rocket example used in the documentation overview looks very nice. But I can't get it to work the same way. 

https://developers.google.com/ar/develop/java/light-estimation/developer-guide-java

- From where should the update method be called?
- How should I apply the setDirectionLightValues?
- How should I apply the setAmbientSphericalHarmonicsLightValues?
- How should I apply the UploadToTexture?

**Comment by chnouman** [comment]
please help.

**Comment by stewe93** [comment]
Right now if you configure the session with `Config.LightEstimationMode.ENVIRONMENTAL_HDR` the whole model becomes black. If I set the default `Sun` node direction, nothing happens, adding a new light `Node` doesn't do anything neither. And how to process the 3 values in this array: 
`float[] intensity = lightEstimate.getEnvironmentalHdrMainLightIntensity();` 
The direction is obvious, of course, the 3 values of intensity isn't, for me at least. Oh and a small thing to mention: `ArImage` is missing from the documentation:  https://developers.google.com/ar/reference/java/arcore/reference/com/google/ar/core/ArImage

**Comment by hardik-intellify** [comment]
yes, we need help, please let us know how can we implement https://developers.google.com/ar/develop/java/light-estimation/developer-guide-java

**Comment by vortice3D** [comment]
Hi there:

Please complete the samples!

That app.doSomething() lines hurt a lot.

Thanks.

**Comment by nvasilescu** [comment]
same problem. Complete your documents. app.something() says a lot.
